modules/system-disaster-recovery/src/main/java/org/apache/ignite/internal/disaster/system/SystemDisasterRecoveryStorage.java

```diff
@@ -52,7 +52,7 @@ boolean isNodeInitialized() {
     }
 
     void markNodeInitialized() {
-        vault.put(new ByteArray(NODE_INITIALIZED_VAULT_KEY), new byte[]{1});
+        vault.put(new ByteArray(NODE_INITIALIZED_VAULT_KEY), new byte[0]);
```

Comment:
We have a constant for empty arrays (see `ArrayUtils.BYTE_EMPTY_ARRAY`), I also suggest to use empty arrays in tests